e2e_test.go

```diff
@@ -34,7 +34,7 @@ func TestEndToEndTrustlessGatewayDomains(t *testing.T) {
 	cancel()
 	rainbow := filepath.Join(runner.Dir, "rainbow")
 
-	args := testcmd.Args(rainbow, "--trustless-gateway-domains", "example.org")
+	args := testcmd.Args(rainbow, "--trustless-gateway-domains", "example.org", "--blockstore", "pebble")
```

Comment:
I do not think much is gainer by testing a matrix of datastores in the current e2e test. I think  Maybe in the future when the e2e test does more than starting and stopping rainbow and checking that is sees some environ vars.

It would be much more interesting to have benchmarks to make comparisons between different datastores.